install_dev_repos.py

```diff
@@ -15,7 +15,7 @@
 from pathlib import Path
 from subprocess import check_output
 
-from importlib_metadata import PackageNotFoundError, distribution
+from importlib.metadata import PackageNotFoundError, distribution
```

Comment:
@ccordoba12, do we need a Python switch here?

```python
if sys.version_info < (3, 10):
    from importlib_metadata import PackageNotFoundError, distribution
else:
    from importlib.metadata import PackageNotFoundError, distribution
```

Reply:
I don't understand very well the difference between `importlib_metadata` and `importlib.metadata`, but both `PackageNotFoundError` and `distribution` are available in `importlib.metadata` in Python 3.8 and 3.9.

Reply:
`importlib_metadata` is a backport of `importlib.metadata`

Reply:
To include new features not available in Python 3.8/3.9 `importlib.metadata`?

Reply:
yes exactly. 

But python has been breaking this interface left right and center and not really introducing any meaningful deprecation cycle.

This line of code https://github.com/cupy/cupy/blob/main/cupy/_environment.py#L490

broke my code with importlib_metadata, but i can't recreate in an isolated environment....